src/ai_hawk/llm/llm_manager.py

```diff
@@ -232,6 +250,10 @@ def _create_model(self, config: dict, api_key: str) -> AIModel:
             return OllamaModel(llm_model, llm_api_url)
         elif llm_model_type == GEMINI:
             return GeminiModel(api_key, llm_model)
+        elif llm_model_type == "huggingface":
```

Comment:
This is already defined below. You can remove it.

Reply:
Sure